CONTRIBUTING.md

```diff
@@ -337,6 +337,10 @@ The exercise should consist of, at minimum:
 * A test suite
 * A reference solution that passes the test (see [#reference-solution](#reference-solution))
 
+You will need to add the exercise to `"exercises"` section of the `config.json` file in the track.
+The order in which the exercises are listed there is the order in which they are fetched by default by `exercism fetch`.
+Typically, exercises are ordered by difficulty, unless there is a particular reason to do otherwise.
```

Comment:
xruby puts new exercises at the end.

The reasoning being that new exercises may have weird errors or ambiguities, and it is better that these are encountered first by the (presumably more experienced) programmers who are working on the later problems, in the hope they will raise GitHub issues for them. Once they have a few successful solutions they can be moved to a more appropriate place in the order.